examples/helpers/tiled.rs

```diff
@@ -135,115 +181,159 @@ pub fn process_loaded_maps(
-                            tiled::Orientation::Hexagonal => {
-                                TilemapType::Hexagon(HexCoordSystem::Row)
-                            }
-                            tiled::Orientation::Isometric => {
-                                TilemapType::Isometric(IsoCoordSystem::Diamond)
-                            }
-                            tiled::Orientation::Staggered => {
-                                TilemapType::Isometric(IsoCoordSystem::Staggered)
-                            }
-                            tiled::Orientation::Orthogonal => TilemapType::Square,
-                        };
+                                    let map_type = match tiled_map.map.orientation {
+                                        tiled::Orientation::Hexagonal => {
+                                            TilemapType::Hexagon(HexCoordSystem::Row)
+                                        }
+                                        tiled::Orientation::Isometric => {
+                                            TilemapType::Isometric(IsoCoordSystem::Diamond)
+                                        }
+                                        tiled::Orientation::Staggered => {
+                                            TilemapType::Isometric(IsoCoordSystem::Staggered)
+                                        }
+                                        tiled::Orientation::Orthogonal => TilemapType::Square,
+                                    };
 
-                        let mut tile_storage = TileStorage::empty(map_size);
-                        let layer_entity = commands.spawn_empty().id();
+                                    let mut tile_storage = TileStorage::empty(map_size);
+                                    let layer_entity = commands.spawn_empty().id();
 
-                        for x in 0..map_size.x {
-                            for y in 0..map_size.y {
-                                let mut mapped_y = y;
-                                if tiled_map.map.orientation == tiled::Orientation::Orthogonal {
-                                    mapped_y = (tiled_map.map.height - 1) - y;
-                                }
+                                    for x in 0..map_size.x {
+                                        for y in 0..map_size.y {
+                                            let mut mapped_y = y;
+                                            if tiled_map.map.orientation
+                                                == tiled::Orientation::Orthogonal
+                                            {
+                                                mapped_y = (tiled_map.map.height - 1) - y;
+                                            }
 
-                                let mapped_x = x as usize;
-                                let mapped_y = mapped_y as usize;
+                                            let mapped_x = x as usize;
```

Comment:
`mapped_x` and `mapped_y` are seemingly only ever used as `i32, so the `as i32` might as well be here.